fastlane/lib/fastlane/actions/git_default_remote_branch.rb

```diff
@@ -0,0 +1,51 @@
+module Fastlane
+  module Actions
+    class GitDefaultRemoteBranchAction < Action
+      def self.run(params)
+        Actions.git_default_remote_branch_name || ""
```

Comment:
Hi, one small **humble request**...Please add unit tests for this action 🙇‍♂️
One of the reasons the Fastlane team doesn't allow new action because it adds lots of maintenance and unit tests really help to reduce the maintenance workload and any enhancement work.

You can add unit tests for all 4 scenarios, you mentioned in the PR description 😇
Thank you!

Reply:
No worries. I have added tests for verifying the command and the first two test cases. However, I have not been able to get the remote working just yet so it may take some time.

Reply:
I added tests for all the conditions. However, this is the first time I have written anything like this in ruby so I probably got something wrong. If so, I'd be happy to sort it out if someone can point me in the right direction.